corehq/apps/cloudcare/templates/cloudcare/partials/menu/persistent_menu.html

```diff
@@ -1,35 +1,32 @@
 {% load i18n %}
 
 <script type="text/template" id="persistent-menu-template">
-  <a type="button"
+  <a id="persistent-menu-arrow-toggle"
+     type="button"
      href="#"
      tabindex="1"
-     data-bs-toggle="offcanvas"
      data-bs-target="#persistent-menu-container"
-     aria-expanded="false"
+     aria-expanded="true"
      aria-controls="persistent-menu-container"
      class="position-fixed top-50 start-0 rounded-end persistent-menu-toggle">
     <i class="fa fa-chevron-right"></i>
   </a>
-  <div id="persistent-menu-container" class="offcanvas offcanvas-start w-auto" tabindex="-1" aria-labelledby="persistent-menu-label">
-    <div class="offcanvas-header persistent-menu-max-width d-flex align-items-center justify-content-between">
+  <div id="persistent-menu-container" class="bg-white h-100 position-relative overflow-scroll"
+    aria-labelledby="persistent-menu-label">
+    <button type="button" id="close-button" class="btn-close text-reset position-absolute"
```

Comment:
`#close-button` might be too generic a term, risking conflict with other elements on the page